<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIfight</class>
 <widget class="QWidget" name="GUIfight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1371</width>
    <height>803</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GUIfight</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="playerPokLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pcPokLabel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VSlabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>450</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>100</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">COLOR:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69))</string>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerAttack">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pcAttack">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>220</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>1311</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>300</number>
    </property>
    <item>
     <widget class="QProgressBar" name="playerhp">
      <property name="font">
       <font>
        <family>华光综艺_CNKI</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="pchp">
      <property name="font">
       <font>
        <family>华光综艺_CNKI</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="chuzhan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光胖头鱼_CNKI</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请选择出战精灵</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="duishou">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>130</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光胖头鱼_CNKI</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请选择对手精灵</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="playerthisPok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Check (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playerlastPok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Back (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playernextPok">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Play (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pcnextPok">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Play (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pcthisPok">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Check (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pclastPok">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>730</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Back (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1371</width>
     <height>803</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1371</width>
     <height>803</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1371</width>
     <height>803</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pokemonClientGUI.qrc">:/pokpic/back.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerlv">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>670</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光胖头鱼_CNKI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pclv">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>670</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光胖头鱼_CNKI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/buttons/Exit (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>VSlabel</zorder>
  <zorder>playerthisPok</zorder>
  <zorder>playerlastPok</zorder>
  <zorder>playernextPok</zorder>
  <zorder>pcnextPok</zorder>
  <zorder>pcthisPok</zorder>
  <zorder>pclastPok</zorder>
  <zorder>exit</zorder>
  <zorder>playerAttack</zorder>
  <zorder>pcAttack</zorder>
  <zorder>chuzhan</zorder>
  <zorder>pcPokLabel</zorder>
  <zorder>playerPokLabel</zorder>
  <zorder>duishou</zorder>
  <zorder>pclv</zorder>
  <zorder>playerlv</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pokemonClientGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>playerlastPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>playernextPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>chuzhan</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>playerthisPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>pclastPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>pcnextPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>1230</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>duishou</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>1060</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>pcthisPok</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>1050</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>playerlv</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>pclv</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>1210</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerthisPok</sender>
   <signal>clicked()</signal>
   <receiver>GUIfight</receiver>
   <slot>okplayerSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>814</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pcthisPok</sender>
   <signal>clicked()</signal>
   <receiver>GUIfight</receiver>
   <slot>okpcSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>785</y>
    </hint>
    <hint type="destinationlabel">
     <x>1017</x>
     <y>852</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>GUIfight</receiver>
   <slot>returnSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>-32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okplayerSlot()</slot>
  <slot>okpcSlot()</slot>
  <slot>returnSlot()</slot>
 </slots>
</ui>
